app/src/common/constants.ts

```diff
@@ -76,7 +76,5 @@ export const LINK_COOKIE_POLICY = process.env.REACT_APP_LINK_COOKIE_POLICY || fa
 export const LINK_FAQ = process.env.REACT_APP_LINK_FAQ || false
 export const DISCLAIMER_TEXT = process.env.REACT_APP_DISCLAIMER_TEXT || false
 
-export const GOOGLE_ANALYTICS_ID = process.env.REACT_APP_GOOGLE_ANALYTICS_ID || null
```

Comment:
You can also remove `LINK_COOKIE_POLICY`

Reply:
`react-ga` dependency could be removed too.

Reply:
Thank you, all removed!